### DISCUSSION TITLE
Update Bootscore ?

### DISCUSSION THREAD

**Discussion by KatyPurry** [bootscore/bootscore#752] [Q&A]
Hello,
I'm running Bootscore version 5.2.0.0. with a child theme.  I'd like to update it. What should I watch out for?  From what I've read on your Blog, there doesn't seem to be any compatibility problem between my version and the latest version. 

The child theme has been modified quite a bit in terms of CSS and a little in terms of code. How do you suggest I proceed?

Thank you very much

Update : read https://github.com/bootscore/bootscore/releases seems a good start !

**Comment by crftwrk** [comment]
Hello, Katy,

there should be no issues updating 5.2 to latest 5.3.x. Most changes have been made in Woo area and new Bootstrap 5.3 colors and yes, [release](https://github.com/bootscore/bootscore/releases) page is where all changes are listed. But please, backup your site and test your child on a staging site first.

**Reply by KatyPurry** [reply]
Thank you for your reply. I'll take note.
